Source/DafnyDriver/DafnyDriver.csproj

```diff
@@ -14,6 +14,10 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
   </PropertyGroup>
+
+  <PropertyGroup Condition="$(RUNTIME_IDENTIFIER) != ''">
```

Comment:
Could you add a comment explaining why we need this? Maybe add a reference to the issue in .NET that causes it: https://github.com/dotnet/sdk/issues/10566

Reply:
Yes that would be helpful for maintainability.